trcli/data_classes/dataclass_testrail.py

```diff
@@ -40,14 +40,16 @@ def calculate_status_id_from_junit_element(junit_result: list) -> int:
         3 - Untested
         4 - Retest
         5 - Failed
+        7 - Automation Passed
+        8 - Automation Failed
         """
         if len(junit_result) == 0:
-            return 1
+            return 7
         test_result_tag = junit_result[0]._tag.lower()
         if test_result_tag == "skipped":
             return 4
         elif test_result_tag == "error" or "failure":
```

Comment:
Might as well fix this OR logic bug while you are at it.

Reply:
Good call